[repo2docker](https://github.com/jupyterhub/repo2docker) failed to build <failed-repository-url>.

## What is the current bug behavior?

<!-- Paste the build logs -->

## What is the expected correct behavior?

repo2docker builds container successfuly.

## Steps to reproduce

1. Visit https://notebooks.gesis.org/binder/
2. Fill `URL` with <failed-repository-url>
3. Click `launch`

## Reproducible at ovh.mybinder.org

1. Visit https://ovh.mybinder.org/
2. Fill `URL` with <failed-repository-url>
3. Click `launch`

resulted in <!-- same issue / different issue / no issue -->.
